modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2806,13 +2716,20 @@ def _groupby_dict_reduce(
         """
         map_dict = {}
         reduce_dict = {}
+        kwargs.setdefault(
+            "default_to_pandas_func",
+            lambda grp, *args, **kwargs: grp.agg(agg_func, *args, **kwargs),
+        )
+
         rename_columns = any(
             not isinstance(fn, str) and isinstance(fn, Iterable)
             for fn in agg_func.values()
         )
         for col, col_funcs in agg_func.items():
             if not rename_columns:
-                map_dict[col], reduce_dict[col] = groupby_reduce_functions[col_funcs]
+                map_dict[col], reduce_dict[col], _ = GroupbyReduceImpl.get_impl(
+                    col_funcs
```

Comment:
what if `col_funcs` describes multiple functions (like a tuple of them)? if it shouldn't, then it seems the `col_funcs` name is wrong

Reply:
`col_funcs` may describe multiple functions (apparently that's why the 'funcs' in the parameter's name is plural), however, in this particular branch `cols_funcs` always describe a single function as we're under `if not rename_columns` condition, which basically means the same (check on how this flag was computed above)